client/snapper.cc

```diff
@@ -468,7 +476,7 @@ list_from_one_config(ProxySnapper* snapper, ListMode list_mode, bool show_used_s
     if (list_mode != LM_ALL && list_mode != LM_SINGLE)
 	show_used_space = false;
 
-    if (show_used_space)
+    if (show_used_space && rescan_qgroups)
```

Comment:
AFAIS this breaks for btrfs if quota is not enabled or LVM since the try/catch is also used to disable show_used_space if it is not supported. So using the new option will cause an error in these cases.

Reply:
Yes, indeed it broke in those cases. Instead of abusing some other method to force disable the used space display, I've extended the Snapper interface to explicitly check for quota support.